tareas/clase-5/calculo-videos.js

```diff
@@ -1,18 +1,56 @@
 const $botonCalcular = document.querySelector("#boton-calcular");
-const $horasPorClase = document.querySelectorAll(".horas");
-const $minutosPorClase = document.querySelectorAll(".minutos");
-const $segundosPorClase = document.querySelectorAll(".segundos");
-const $textoResultadoTotal = document.querySelector("#resultado");
+const $botonAgregarClase = document.querySelector("#boton-agregar-clase");
 
+function agregarClase(numeroDeClase) {
+    const $formClases = document.querySelector("#form-clases");
 
+    const $h4 = document.createElement("h4");
+    $h4.textContent = `Clase ${numeroDeClase}`;
+
+    const $div = document.createElement("div");
+    $div.id = `video ${numeroDeClase}`;
+
+    const $inputHoras = document.createElement("input");
+    $inputHoras.type = "number"
+    $inputHoras.placeholder = "Ingrese las horas de video";
+    $inputHoras.className = "horas";
+
+    const $inputMinutos = document.createElement("input");
+    $inputMinutos.type = "number";
+    $inputMinutos.placeholder = "Ingrese los minutos de video";
+    $inputMinutos.className = "minutos";
+
+    const $inputSegundos = document.createElement("input");
+    $inputSegundos.type = "number";
+    $inputSegundos.placeholder = "Ingrese los segundos de video";
+    $inputSegundos.className = "segundos";
+
+    $formClases.appendChild($h4);
+    $formClases.appendChild($div);
+    $div.appendChild($inputHoras);
+    $div.appendChild($inputMinutos);
+    $div.appendChild($inputSegundos);
+};
+
+
+$botonAgregarClase.onclick = function () {
+    const numeroDeClase = Number(prompt("Ingrese el número de clase (en números)"));
+
+    agregarClase(numeroDeClase);
+
+    return false;
+}
 
 $botonCalcular.onclick = function () {
+    const $horasPorClase = document.querySelectorAll(".horas");
+    const $minutosPorClase = document.querySelectorAll(".minutos");
+    const $segundosPorClase = document.querySelectorAll(".segundos");
+    const $textoResultadoTotal = document.querySelector("#resultado");
+
     let horasTotalesDeClase = 0;
     let minutosTotalesDeClase = 0;
     let segundosTotalesDeClase = 0;
 
-
-
     for (let i = 0; i < $horasPorClase.length; i++) {
```

Comment:
Todas las clases tienen horas, minutos y segundos, por lo que podrías hacer 1 solo for y hacer la acumulación adentro de ese único for.